toc.yml

```diff
@@ -1,3 +1,6 @@
+- name: Setup
```

Comment:
Ich glaube ich würde das Tutorial eher in Models rein schieben anstatt dafür einen extra Reiter auf zu machen. Das Tutorial hier ist sehr stark auf die Unterstützung von Metamodellen fixiert, für andere Teile von NMF ist das so nicht zwangsläufig notwendig.